### PR TITLE
Avatars

### PR DESCRIPTION
**Pull request by Sameony**
Added different types of avatars
image
image
image

### CODE REVIEW

**Review by shuntzhere — APPROVED** [review]
The code looks clean, It's nice that you have created utilities for spacings and margin paddings.

File: Templates/Avatars/avatar.html [hunk 1]
```diff
@@ -127,13 +127,13 @@ <h2 class="contentHeader mg-y-sm">Square Avatars</h2>
         </div>
         <pre>
             <code class="showcase">
-                <div class="cardShow">
-            <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-xs mg-md">
-            <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-s mg-md">
-            <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-m mg-md">
```
Comment (vaibhav-mahalle) [inline Templates/Avatars/avatar.html]: here are  you using md and m both for representing medium size?

File: Templates/Avatars/avatar.html [hunk 2]
```diff
@@ -127,13 +127,13 @@ <h2 class="contentHeader mg-y-sm">Square Avatars</h2>
...
-            <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-l mg-md">
-            <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-xl mg-md">
-                </div>
+    <div class="cardShow">
+        <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-xs mg-md">
+        <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-s mg-md">
+        <img src="/Assets/Images/avatar2.jpg" alt="squareAvatar" class="avatar-m mg-md">
```
Comment (vaibhav-mahalle) [inline Templates/Avatars/avatar.html]: same as above.

**Review by vaibhav-mahalle — APPROVED** [review]
everything else looks good. suggested some changes.

File: Templates/Avatars/avatar.html (hunk 1 above)
Comment (Sameony) [reply Templates/Avatars/avatar.html]: ah, yeah
i've used two digits for white spacings such as md-sm-xl etc for paddings and margins
and single digits for letters and elements sizes, such as font weight and font sizes. if it's feeling inconsistent, let me know, i'll switch it up

File: Templates/Avatars/avatar.html (hunk 2 above)
Comment (Sameony) [reply Templates/Avatars/avatar.html]: updated in the latest issue, thanks for pointing it out

**Pull request merged**